### PR TITLE
DAOS-17852 test: CR Automate Test - Check repair corner case

### PR DESCRIPTION
**Pull request by shimizukko**
1. Create a pool.
2. Inject orphan pool fault.
3. Start the checker with interactive mode.
4. Verify that the orphan pool is detected.
5. Repair with invalid ID. Verify error message.
6. Repair with invalid action. Verify error message.
7. Repair with correct ID and action.
8. Repair again. Verify error message.

Skip-unit-tests: true
Skip-fault-injection-test: true
Test-tag: test_check_repair_corner_case

### Steps for the author:

* [ ] Commit message follows the [guidelines](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Appropriate [Features or Test-tag](https://daosio.atlassian.net/wiki/spaces/DC/pages/10984259629/Test+Tags) pragmas were used.
* [ ] Appropriate [Functional Test Stages](https://daosio.atlassian.net/wiki/spaces/DC/pages/12147556353/CI+Functional+Test+Stages) were run.
* [ ] At least two positive code reviews including at least one code owner from each category referenced in the PR.
* [ ] Testing is complete. If necessary, forced-landing label added and a reason added in a comment.

#### After all prior steps are complete:
* [ ] Gatekeeper requested (daos-gatekeeper added as a reviewer).

### CODE REVIEW

File: src/tests/ftest/recovery/check_repair.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        #    "Discard the pool",
+        #    "Ignore the pool finding"
+        # ]
+        # We want to select "Re-add the pool", so action is 0.
+        act_msgs = query_reports[0]["act_msgs"]
+        action = None
+        for i, act_msg in enumerate(act_msgs):
+            if act_msg == "Re-add the pool":
+                action = str(i)
+                break
```
Comment (daltonbohning) [inline src/tests/ftest/recovery/check_repair.py]: FYI there is a `list.index(str)` that does this for you. E.g.
```
>>> l = ["a", "b", "c"]
>>> l.index("b")
1
>>> l.index("d")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: 'd' is not in list
```

So you could do
```suggestion
        try:
            action = str(act_msgs.index("Re-add the pool"))
        except ValueError:
            self.fail(...)
```

**Review by daltonbohning — DISMISSED** [review]

File: src/tests/ftest/recovery/check_repair.py (hunk 1 above)
Comment (shimizukko) [reply src/tests/ftest/recovery/check_repair.py]: Done. Thanks.

**Review by daltonbohning — APPROVED** [review]

**Review by phender — APPROVED** [review]
